### PR TITLE
Manifest and VLAN changes that happened during the downtime

### PR DESCRIPTION
**Pull request by hakasapl**
This PR includes:
* Switch manifest changes
* Added a "all" feature to the dell os9 module for tagged vlans
* Split vlan file into MOC/OCT
* Removed OCT5 switches

@naved001 could you look at the `group_vars/moc/vlans.yml` file? These are all the VLANs on all MOC switches (not OCT). I've removed any VLAN that wasn't being used in the network, but it's possible I over trimmed this if some things are local to the Ciscos only. The `group_vars/all/vlans.yml' still exists and has all the original VLANs, if you want to reference it.

### CODE REVIEW

File: group_vars/moc/vlans.yaml [hunk 1]
```diff
@@ -0,0 +1,940 @@
+vlans:
+  10:
+    name: "CSAIL-MAIN"
+    description: "Various Openshift clusters"
+  105:
+    name: "MOC-BU-PUBLIC"
+    description: "Public IPs from BU"
+  127:
+    name: "MOC-NEU-PUBLIC"
+    description: "NEU Public IP for infrastructure"
```
Comment (naved001) [inline group_vars/moc/vlans.yaml:10]: VLAN 201 is missing. It's where the foreman instance is running and we still have a bunch of VMs and oVirt hosts on that network.

File: group_vars/moc/vlans.yaml [hunk 2]
```diff
@@ -0,0 +1,940 @@
...
+  204:
+    name: "MOC-GENERAL"
+    description: "Intranet (routable to internet). SNMP for client nodes."
```
Comment (naved001) [inline group_vars/moc/vlans.yaml:13]: VLAN 205 should also be here https://github.com/CCI-MOC/ansible-switches/blob/main/group_vars/all/vlans.yaml#L33

File: group_vars/moc/vlans.yaml [hunk 3]
```diff
@@ -0,0 +1,940 @@
...
+    description: "New England Storage Exchange (NESE)"
+  213:
+    name: "MOC-PROD-CEPH-ISCSI"
+    description: "Ceph Cluster iSCSI"
+  214:
+    name: "MOC-NEW-ESI"
+    description: "New ESI deployment"
+  250:
+    name: "MOC-PROD-CEPH"
+    description: "Ceph public (for clients)"
```
Comment (naved001) [inline group_vars/moc/vlans.yaml:28]: VLAN 277 and 280 should be here - those are managed by MaaS. 

Not sure if you skipped 278 on purpose (iventoy)

Comment (hakasapl) [reply group_vars/moc/vlans.yaml:34]: I skipped iventoy on purpose, we can circle back if we ever end up using it

**Comment by hakasapl** [comment]
I've added back those vlans

**Pull request merged**